sdk/lib/drivers/sound/mmixer/mixer.c

```diff
@@ -818,10 +818,9 @@ MMixerInitialize(
      MixerContext->MixerContext = (PVOID)MixerList;
 
     /* start enumerating all available devices */
-    Count = 0;
     DeviceIndex = 0;
 
-    do
+    for (;;)
```

Comment:
```suggestion
    for (DeviceIndex = 0; ; DeviceIndex++)
```

Reply:
Done, thanks. Moved `DeviceIndex` intialization inside `for(;;)` loop block and also updated a comment a bit, to make it corresponding better to the current behaviour.